QuizApp/Customs/PopupButton.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  PopupButton.swift
+//  QuizApp
+//
+//  Created by Tamuna Kakhidze on 18.11.24.
+//
+
+import UIKit
+
+final class PopupButton: UIButton {
+    
+    // MARK: - Init
+    init() {
+        super.init(frame: .zero)
+        initialize()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: - Configurations
+    private func initialize() {
+        setAppearance()
+        setConstraints()
+    }
+    
+    private func setAppearance() {
+        self.backgroundColor = Constants.Colors.backgroundColor
+        self.clipsToBounds = true
+        self.contentHorizontalAlignment = .center
+        self.contentVerticalAlignment = .center
```

Comment:
I think contentHorizontalAlignment is by default centre, but not sure. please check for contentVerticalAlignment as well